cruise-control-client/tests/client/Endpoint.py

```diff
@@ -0,0 +1,26 @@
+import argparse
```

Comment:
I've provided some tests to aide in maintaining compatibility through the refactoring that I have done here, but I have not modified the circleci configurations to have them run.